completion/available/fabric.completion.bash

```diff
@@ -79,9 +79,6 @@ function __fab_fabfile_mtime() {
 # Completion for "fab" command
 #
 function __fab_completion() {
-    # Return if "fab" command doesn't exists
-    [[ -e `which fab 2> /dev/null` ]] || return 0
```

Comment:
Did you forget to replace this check?

Reply:
No, as I said in the commit message: once were already *inside* the completion handler, there's no point to testing if the command is installed!

Reply:
I did not see the commit message for this particular file while reviewing. Upon deeper review, it looks like this file is actually a candidate for vendoring, as it includes a license. I think we should drop this file from the PR for now and open an issue to deal with this file later.